webapp/Connector/src/view/MabGrid.js

```diff
@@ -0,0 +1,617 @@
+                column.getEl().removeCls('x-column-header-over');
+            }
+        });
+
+        var filters = Connector.getState().getMabFilters(true);
+        Ext.each(filters, function(filter)
+        {
+            var f = filter.gridFilter[0];
+            var fieldName = f.getColumnName();
+            var colIndexes = Connector.view.MabGrid.ColumnMap[fieldName];
+            if (colIndexes && Ext.isArray(colIndexes)) {
+                Ext.each(colIndexes, function(colIndex) {
+                    var col = grid.headerCt.getHeaderAtIndex(colIndex);
+                    if (col)
+                    {
+                        col.getEl().addCls('filtered-column');
+                    }
+                })
+            }
+        }, this);
+    },
+
+
+    triggerColumnHeaderOver: function(event, item, options) {
+        var colInd = options.colInd;
+        var isEnter = options.isEnter;
+        var colHeader = Ext.DomQuery.select("div.x-column-header.mabcolheader" + colInd)[0];
+        var highlightCls = 'x-column-header-over';
+        if (colHeader) {
+            if (isEnter) {
+                if (colHeader.className.indexOf(highlightCls) === -1) {
+                    colHeader.className += ' ' + highlightCls;
+                }
+            }
+            else {
+                colHeader.className = colHeader.className.replace(' ' + highlightCls, '');
+            }
+        }
+    },
+
+    onViewResize : function() {
+        Ext.defer(function() {
+            if (this.getModel().isActive())
+            {
+                if (this.grid)
+                {
+                    var size = this.getWidthHeight();
+                    this.getGrid().setSize(size.width, size.height);
+                }
+            }
+        }, 50, this);
+    },
+
+    onActivate : function()
+    {
+        this.getModel().setActive(true);
+        var infopane = Ext.ComponentQuery.query('app-main > #eastview > #navfilter > #filterstatuscontainer');
+        if (infopane && infopane.length > 0)
+            infopane[0].hide();
+    },
+
+    onDeactivate : function()
+    {
+        this.getModel().setActive(false);
+        var infopane = Ext.ComponentQuery.query('app-main > #eastview > #navfilter > #filterstatuscontainer');
+        if (infopane && infopane.length > 0)
+            infopane[0].show();
+    },
+
+    getModel : function() {
+        return this.model;
+    },
+
+    getWidthHeight : function() {
+        var box = this.getBox();
+
+        return {
+            width: box.width - 27,
+            height: box.height - this.headerHeight + this.titleHeight
+        };
+    },
+
+    cellRenderer : function(v, meta, record, measure) {
+        return v;
+    },
+
+    requestExportCSV: function() {
+        this.requestExport(false);
+    },
+
+    requestExportExcel: function() {
+        this.requestExport(true);
+    },
+
+    requestExport : function(isExcel) {
+        //
```

Comment:
Is this just a placeholder for a future story?
Is the button in this case just disabled for now? If not, probably want to either disable it or give the user a message to the effect of "not yet implemented / coming soon".

Reply:
Yeah it's just a placeholder. The button is hidden for now and will be enabled in a future story.